src/debugger/server/Debugger.js

```diff
@@ -291,6 +297,41 @@ export class DebugServer {
     return false;
   }
 
+  /*
+    Displays PP error message, then waits for user to run the program to
+    continue (similar to a breakpoint).
+  */
+  handlePrepackError(diagnostic: CompilerDiagnostic) {
+    invariant(diagnostic.location && diagnostic.location.source);
+    let location = diagnostic.location;
+    let message = `${diagnostic.severity} ${diagnostic.errorCode}: ${diagnostic.message}`;
+    this._channel.sendStoppedResponse(
+      "Diagnostic",
+      location.source || "",
+      location.start.line,
+      location.start.column,
+      message
+    );
+    // No ast parameter b/c you cannot stepInto/Over a line that's causing an exception
+    this.waitForRun();
+  }
+
+  // Return whether the debugger should stop on a CompilerDiagnostic of a given severity.
+  evaluateDiagnosticSeverity(severity: Severity): boolean {
```

Comment:
I would call it "shouldStopForSeverity()" to clearly indicate this function decides whether debugger should stop or not.